### PR TITLE
f-content-cards@1.9.0: Control styling changes from props

### PR DESCRIPTION
**Pull request by asherstoppard**
Remove explicit card type checks from `CardContainer` and move them to the appropriate components.

## UI Review Checks

- **N/A** README and/or UI Documentation has been [created|updated]
- **N/A** Unit tests have been [created|updated]
- **N/A** This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

### CODE REVIEW

**Review by minrwhite — DISMISSED** [review]
💯

File: packages/f-content-cards/src/components/cardTemplates/VoucherCard.vue [hunk 1]
```diff
@@ -43,6 +44,12 @@ export default {
         };
     },
 
+    computed: {
+        isAnniversaryCard () {
+            return this.cardType === 'Anniversary_Card_1';
```
Comment (minrwhite) [inline packages/f-content-cards/src/components/cardTemplates/VoucherCard.vue]: I think you need to account for the transformation done by normaliseCardType here:
```suggestion
            return this.cardType === 'anniversaryCard1';
```

**Comment by minrwhite** [comment]
I like the change in approach to where these per-card type customisations are encapsulated - been trying to work out what is causing the percy differences, and think I've got one of the issues, but not sure about the POCC change yet.

**Comment by asherstoppard** [comment]
@benwhite-deltas Just committed that change, I'll take a look through the Percy screens once built and see if I can work through the issues :+1:

**Review by minrwhite — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**